### PR TITLE
Docs: Clean up of 'acl' module docs

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This is part of a series of module doc cleanups.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
acl

##### ANSIBLE VERSION
v2.8

### CODE REVIEW

File: lib/ansible/modules/files/acl.py [hunk 1]
```diff
@@ -99,14 +96,14 @@
...
-- name: Removes the acl for Joe on a specific file
+- name: Removes the ACL for Joe on a specific file
   acl:
     path: /etc/foo.conf
     entity: joe
     etype: user
     state: absent
 
-- name: Sets default acl for joe on foo.d
+- name: Sets default ACL for joe on foo.d/
```
Comment (samccann) [inline lib/ansible/modules/files/acl.py]: remove stray / at the end...

**Review by samccann — DISMISSED** [review]

File: lib/ansible/modules/files/acl.py (hunk 1 above)
Comment (dagwieers) [reply lib/ansible/modules/files/acl.py]: Well, without the trailing /, it is unclear this is a directory.
I think it is good practice to keep a trailing slash for directories (even when they are strictly no needed).

**Comment by dagwieers** [comment]
Please re-review !

File: lib/ansible/modules/files/acl.py [hunk 2]
```diff
@@ -10,87 +11,83 @@
...
+    - The full path of the file or object.
     aliases: [ name ]
-    required: true
-
+    required: yes
   state:
     description:
-      - defines whether the ACL should be present or not.  The C(query) state gets the current acl without changing it, for use in 'register' operations.
+    - Define whether the ACL should be present or not.
+    - The C(query) state gets the current ACL without changing it, for use in 'register' operations.
```
Comment (acozine) [inline lib/ansible/modules/files/acl.py]: Should this be C(register)?

File: lib/ansible/modules/files/acl.py [hunk 3]
```diff
@@ -10,87 +11,83 @@
...
     type: bool
-    default: 'yes'
-
+    default: yes
   default:
     description:
-      - if the target is a directory, setting this to yes will make it the default acl for entities created inside the directory. It causes an error if
-        path is a file.
+    - If the target is a directory, setting this to C(yes) will make it the default ACL for entities created inside the directory.
+    - It causes an error if path is a file.
```
Comment (acozine) [inline lib/ansible/modules/files/acl.py]: Now that there are two lines, "it" is a bit unclear. I'd change this to "Setting C(default) to C(yes) causes an error if the path is a file."

File: lib/ansible/modules/files/acl.py [hunk 4]
```diff
@@ -10,87 +11,83 @@
...
+    - DEPRECATED.
+    - The ACL to set or remove.
+    - This must always be quoted in the form of C(<etype>:<qualifier>:<perms>).
+    - The qualifier may be empty for some types, but the type and perms are always required.
+    - C(-) can be used as placeholder when you do not care about permissions.
+    - This is now superseded by entity, type and permissions fields.
   recursive:
     description:
-      - Recursively sets the specified ACL (added in Ansible 2.0). Incompatible with C(state=query).
+    - Recursively sets the specified ACL (added in Ansible 2.0).
```
Comment (acozine) [inline lib/ansible/modules/files/acl.py]: Since we have a line for `version_added`, I'd remove the "(added in Ansible 2.0)".

**Review by acozine — DISMISSED** [review]
A great improvement overall. I noticed a couple of things - once those are addressed, this one's ready to merge.

File: lib/ansible/modules/files/acl.py (hunk 2 above)
Comment (dagwieers) [reply lib/ansible/modules/files/acl.py]: Fixed !

File: lib/ansible/modules/files/acl.py (hunk 3 above)
Comment (dagwieers) [reply lib/ansible/modules/files/acl.py]: Indeed, much better !

File: lib/ansible/modules/files/acl.py (hunk 4 above)
Comment (dagwieers) [reply lib/ansible/modules/files/acl.py]: Good catch !

**Comment by dagwieers** [comment]
Changes made, please re-review !

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
rebuild_merge

**Pull request merged**